README.md

```diff
@@ -110,15 +109,13 @@ app$callback(
             ),
         layout = list(title = newTitle)
     )
-  },
-  output("giraffe", "figure")
+  }
 )
 
-app$callback(
-  function(x = input("graphTitle"), y = input("graphTitle", "type")) {
+app$callback(output=list("outputID"), inputs=list(list(id="graphTitle", property="type")), user_function=
+  function(x, y) {
     sprintf("You've entered: '%s' into a '%s' input control", x, y)
-  },
-  output("outputID")
+  }
```

Comment:
similarly down here

Reply:
Oh I see, the original callbacks here actually did have two inputs. Perhaps we can use that in at least one of these cases to show how to use `state`:
```R
app$callback(output("outputID", "children"),
  list(input("graphTitle", "value"), state("graphTitle", "type")),
  function(x, y) {
    sprintf("You've entered: '%s' into a '%s' input control", x, y)
  }
)
```